server/http_test.go

```diff
@@ -51,7 +51,7 @@ func (c *fakeConnector) TrustedEmailProvider() bool {
 
```

Comment:
It might be worth a test case demonstrating a rejection when an inbound request attempts to submit with `register=1` and handleAuthFunc has been called with `registrationEnabled=false`